romancal/conftest.py

```diff
@@ -41,8 +41,19 @@ def slow(request):
     return request.config.getoption("--slow")
 
 
+@pytest.fixture(scope="function")
+def function_jail(tmp_path):
+    """Perform test in a pristine temporary working directory."""
+    old_dir = os.getcwd()
+    os.chdir(tmp_path)
+    try:
+        yield str(tmp_path)
+    finally:
+        os.chdir(old_dir)
+
+
 @pytest.fixture(scope="module")
-def jail(request, tmpdir_factory):
```

Comment:
This `tmpdir_factory` usage was missed in the earlier PR likely because it is only used in regtests.